webapp/channels/src/components/claim/components/oauth_to_email.tsx

```diff
@@ -3,14 +3,12 @@
 
 import classNames from 'classnames';
 import React, {useRef, useState} from 'react';
-import {FormattedMessage} from 'react-intl';
+import {FormattedMessage, useIntl} from 'react-intl';
 
 import type {AuthChangeResponse} from '@mattermost/types/users';
 
 import {oauthToEmail} from 'actions/admin_actions.jsx';
 
-import LocalizedInput from 'components/localized_input/localized_input';
-
 import Constants from 'utils/constants';
 import {t} from 'utils/i18n';
```

Comment:
is this import still needed?

Reply:
no, I quit it

Reply:
@M-ZubairAhmed I quit already the `t` import

Reply:
i see that now thank you